### DISCUSSION TITLE
What are the estimated Android/iOS system requirements at the moment?

### DISCUSSION THREAD

**Discussion by Saraistupid** [ppy/osu#30660] [Q&A] [answered]
Are there any estimated system requirements for Android/iOS? The game runs really badly on both my iPhone X and Samsung Tab A7 (2020)(which is low spec I know) and it should run better at a better fps than the 10-40fps range (weirdly enough, both vastly different devices perform *the exact same*, just less fps frequent drops on the iPhone X) on either machines considering what this game is

**Comment by Joehuu** [comment, marked as answer]
I have a Pixel 6a and iPad 9th gen and they seem to handle lazer well at constant 60 with some dips, but the aim is making every device (at least on iOS) run well.

Note that mobile is not a priority, but performance in general is a constant work in progress.

**Reply by Saraistupid** [reply]
So current requirement is basically a 2021-2022 midrange or a 2019 flagship, alright, thank you!
